src/components/InputV2/Base.tsx

```diff
@@ -76,7 +79,8 @@ export const BaseInput = ({
   </Label>
 )
 
-export interface BaseInputProps {
+export interface BaseInputProps
+  extends Omit<InputHTMLAttributes<HTMLInputElement>, 'onChange'> {
```

Comment:
Here i omit the `onChange` type because we override it with our own version of `onChange`